src/renderers/shaders/ShaderChunk/clipping_planes_fragment.glsl

```diff
@@ -0,0 +1,11 @@
+#if NUM_USER_CLIPPING_PLANES > 0
+
+	for ( int i = 0; i < NUM_USER_CLIPPING_PLANES; ++ i ) {
+
+		vec4 plane = userClippingPlanes[ i ];
+		if ( dot( vViewPosition, plane.xyz ) > plane.w ) discard;
```

Comment:
OpenGL ES has an extension from Apple that adds clipping planes:

https://www.khronos.org/registry/gles/extensions/APPLE/APPLE_clip_distance.txt

I think glClipPlane was in openGL 1.x or something like that: https://www.khronos.org/opengles/sdk/1.1/docs/man/glClipPlane.xml

I think it got deprecated in favor of clip distance -- I am not familiar with these OpenGL 4.5 changes: http://web.eecs.umich.edu/~sugih/courses/eecs487/common/notes/APITables.xml